examples/legacy_pip_import/WORKSPACE

```diff
@@ -8,7 +8,7 @@ http_archive(
     sha256 = "e46612e9bb0dae8745de6a0643be69e8665a03f63163ac6610c210e80d14c3e4",
```

Comment:
The version used here would have to be updated at the same time as your change below, otherwise the example is broken.

Reply:
I think the example at head will be broken between this landing and the next release going out, yes. It's too hard to make it compatible such that you can run this example either against a release or against our sources at HEAD, during a breaking change like this. If we want to be really careful about it we should land this on a 0.1.x feature branch instead of master, then after cutting the release we change the branches back. Up to you if it's worth it.

Reply:
Probably not worth it.